crates/virtio-queue/src/lib.rs

```diff
@@ -20,9 +24,14 @@ use vm_memory::{
     Address, ByteValued, Bytes, GuestAddress, GuestAddressSpace, GuestMemory, GuestMemoryError,
 };
 
-pub(super) const VIRTQ_DESC_F_NEXT: u16 = 0x1;
-pub(super) const VIRTQ_DESC_F_WRITE: u16 = 0x2;
-pub(super) const VIRTQ_DESC_F_INDIRECT: u16 = 0x4;
+use log::error;
+
+/// Marks a buffer as continuing via the next field
```

Comment:
We prefer to end the comments with full stop.

Reply:
Done